panels/automated_emails.py

```diff
@@ -12,13 +12,17 @@ def __init__(self, *args, **kwargs):
 PanelAppEmail('Your {EVENT_NAME} Panel Application Has Been Received', 'panel_app_confirmation.txt')
 
 PanelAppEmail('Your {EVENT_NAME} Panel Application Has Been Accepted', 'panel_app_accepted.txt',
-              lambda app: app.status == c.ACCEPTED)
+              lambda app: app.status == c.ACCEPTED,
+              ident='panel_accepted')
```

Comment:
I set idents on these emails, both to make them easier to look up and because we will be giving them unique subjects in a later commit anyway.